db/migrate/20220823123916_add_confirmable_to_devise_v1.rb

```diff
@@ -0,0 +1,4 @@
+class AddConfirmableToDeviseV1 < ActiveRecord::Migration[7.0]
+  def change
+  end
+end
```

Comment:
[OPTIONAL] Since this migration file is empty, it would be better if you removed it.